index.html

```diff
@@ -1343,50 +1355,69 @@
           </p>
           <p>
             When the <code><dfn>close</dfn>()</code> method is called on a
-            <a>PresentationConnection</a>, the user agent MUST run the
-            algorithm to <a data-lt="close-algorithm">close a presentation
-            connection</a> with <a>PresentationConnection</a>, a
-            <code>reason</code> of <code>closed</code>, and an empty
-            <code>message</code>.
+            <a>PresentationConnection</a> <var>S</var>, the <a>user agent</a>
+            MUST run the following steps:
           </p>
+          <ol>
+            <li>If the <a>presentation connection state</a> of <var>S</var> is
```

Comment:
Perhaps these are prerequisites of the "start closing" procedure, i.e. move them below?